src/commands/new-proj.ts

```diff
@@ -58,6 +60,17 @@ async function action() {
     if (pathResult) {
       givenPath = pathResult;
     }
+
+    const awsResult = await this.prompt({
+      type: "list",
+      name: "aws",
+      choices: [ "Yes", "No" ],
+      message: "Attach project to AWS: "
+    });
+
+    if (awsResult.aws) {
+      (awsResult.aws == "Yes") ? aws = true : aws = false;
+    }
```

Comment:
couldnt this be done with Promptutils.confirmPrompt?

Reply:
Replaced with Promptutils.confirmPrompt